### ISSUE TITLE
Changes required for xdebug when using lando with VPN (NordVPN)

### ISSUE THREAD

**Issue by drupov** [lando/lando#2396]
I have a general question: when using my normal internet connection, xdebug with PhpStorm works like a charm (see my setup below).

When I connect to NordVPN the setup does not get it, resulting in my xdebug session totallybeing ignored (same breakpoints do not stop at all).

My `.lando.yml`-file looks like this:

```
name: drupal8
recipe: drupal8
config:
  via: nginx
  php: '7.2'
  webroot: web
  xdebug: true
services:
  appserver:
    overrides:
      environment:
        PHP_IDE_CONFIG: 'serverName=appserver'
    config:
      php: php.ini
tooling:
  drush:
    service: appserver
    cmd: drush --root=/app/web
  drupal:
    service: appserver
    cmd: drupal --root=/app/web
```

What can be the cause here?

**Comment by dustinleblanc** [comment]
Hey @drupov! Not sure here, I don't use Xdebug at all, but my guess is that there is some kind of network shenanigans at play here. Might be good to run a search for docker and NordVPN on the web and throw xdebug in for good measure. I don't imagine Lando is doing much here.

**Comment by drupov** [comment]
Thanks! I'll try to reach out to NordVPN support, Google search I did already. If I find out something, I will update the issue.

**Comment by Renrhaf** [comment]
Hey there @drupov any news about this ?

**Comment by drupov** [comment]
Hey, @Renrhaf - I could not find a solution, but a workaround.

* disable the VPN
* set a breakpoint early in the script (where the VPN is not needed yet)
* enable VPN
* you can proceed after that...

Hope you have at least some success that way.

Cheers

**Comment by Renrhaf** [comment]
Hey @drupov, Thanks !
I also investigated a little bit and traced the root of the issue.

When logged to the VPN (host.docker.internal being the IP xDebug tries to connect to in my case) : 
```
/srv/api # ping host.docker.internal
ping: bad address 'host.docker.internal'
``` 

When not logged, the host is accessible : 
```
/srv/api # ping host.docker.internal
PING host.docker.internal (192.168.65.2): 56 data bytes
64 bytes from 192.168.65.2: seq=0 ttl=37 time=0.597 ms
```

So I managed to work around that issue by disabling the VPN for Docker Desktop application in my VPN app (NordVPN).
![image](https://user-images.githubusercontent.com/690629/109184807-5532f980-775d-11eb-9d2e-e7d7f73d958a.png)

Hope this helps someone ! Have a nice day.
